docs/error-reporting/whats-new.md

```diff
@@ -5,6 +5,22 @@ sidebar_label: What's New
 description: See what's new with the Backtrace Console, including new features and improvements.
 ---
 
+## View Users' Last Login (2022-12-02)
+As an admin, you can now check when a user last logged in to their account.
+
+From the **Organization Settings**, click **Users**. In the list of users, under **Last login**, you'll see the date and time of the user's last login.
+
+If a new user hasn't logged in to their account for the first time or hasn't logged in since August 1st 2022, the column will not be populated.
+
+<img src="/img/error-reporting/whats-new/org-settings-last-login.png" width="400"/>
```

Comment:
Consider cropping this image to just show the first two log ins. I think that is enough to illustrate the feature.